shibboleth/views.py

```diff
@@ -10,7 +10,7 @@
 try:
     from django.utils.six.moves.urllib.parse import quote
 except ImportError:
-    from urllib import quote
+    from urllib.parse import quote  
```

Comment:
Thanks, @MLHale! We've dropped support for python 2, so could you just remove the whole try/except block? We can just use the python3 import.

Reply:
Sure thing.
see af67522